src/file_utils.cr

```diff
@@ -146,6 +146,74 @@ module FileUtils
     end
   end
 
+  # Creates a hard link *new_path* which points to *old_path*.
+  # If *new_path* already exists and is a directory, creates a link *new_path/old_path*.
+  #
+  # ```
+  # FileUtils.ln("/usr/bin/vim", "/usr/bin/emacs")
+  # FileUtils.ln("foo.c", "/tmp")
+  # ```
+  def ln(old_path : String, new_path : String)
+    if Dir.exists?(new_path)
+      File.link(old_path, File.join(new_path, File.basename(old_path)))
+    else
+      File.link(old_path, new_path)
+    end
+  end
+
+  # Creates a hard link to each path in *old_paths* inside the *dest_dir* directory.
+  # If *dest_dir* is not a directory, raises an `ArgumentError`.
+  #
+  # ```
+  # FileUtils.ln(["vim", "emacs", "nano"], "/usr/bin")
+  # ```
+  def ln(old_paths : Enumerable(String), dest_dir : String)
+    raise ArgumentError.new("No such directory : #{dest_dir}") unless Dir.exists?(dest_dir)
+
+    old_paths.each do |path|
+      ln(path, dest_dir)
+    end
+  end
+
+  # Creates a symbolic link *new_path* which points to *old_path*.
+  # If *new_path* already exists and is a directory, creates a link *new_path/old_path*.
+  #
+  # ```
+  # FileUtils.ln_s("/var/log", "logs")
+  # FileUtils.ln_s("src", "/tmp")
+  # ```
+  def ln_s(old_path : String, new_path : String)
```

Comment:
You're not renaming, so *old* and *new* aren't really precise.  Something like *source* and *destination* would be clearer.